### PR TITLE
Update julia.md

### PR DESCRIPTION
**Pull request by logankilpatrick**
Added sample projects to look at for reference.

### CODE REVIEW

**Comment by logankilpatrick** [comment]
Just fixed a quick typo @Pezi777 .

**Comment by logankilpatrick** [comment]
Can I get another review on this @Pezi777

File: user/languages/julia.md [hunk 1]
```diff
@@ -119,3 +119,14 @@ TRAVIS_JULIA_VERSION
 ```
 
 In addition, `JULIA_PROJECT` is set to `@.`, which means Julia will search through parent directories until a `Project.toml` or `JuliaProject.toml` file is found; the containing directory then is used the home project/environment.
```
Comment (Pezi777) [inline user/languages/julia.md]:
```suggestion
In addition, `JULIA_PROJECT` is set to `@.`, which means Julia will search through parent directories until a `Project.toml` or `JuliaProject.toml` file is found; the containing directory then is used in the home project/environment.
```

File: user/languages/julia.md [hunk 2]
```diff
@@ -119,3 +119,14 @@ TRAVIS_JULIA_VERSION
...
+
+## Example Projects
+
+Below are a list of open source Julia projects utilizing Travis CI in different ways:
```
Comment (Pezi777) [inline user/languages/julia.md]:
```suggestion
Here's a list of open-source Julia projects utilizing Travis CI in different ways: 
```

**Comment by logankilpatrick** [comment]
Done!

**Comment by Pezi777** [comment]
thank you so much @logankilpatrick :-)

**Pull request merged**